storage/src/cache/cachedfile.rs

```diff
@@ -36,11 +37,69 @@ use crate::meta::{BlobMetaChunk, BlobMetaInfo};
 use crate::utils::{alloc_buf, copyv, readv, MemSliceCursor};
 use crate::{StorageError, StorageResult, RAFS_DEFAULT_CHUNK_SIZE};
 
+const DOWNLOAD_META_RETRY_COUNT: u32 = 5;
+const DOWNLOAD_META_RETRY_DELAY: u64 = 500;
+
+#[derive(Default, Clone)]
+pub(crate) struct FileCacheMeta {
+    status: Arc<AtomicBool>,
```

Comment:
Do you mind adding a comment for `status`? 
looks like `status` indicates the failure of downloading blob.meta

Reply:
renamed to `has_error`.